grifts/db.go

```diff
@@ -203,17 +203,18 @@ var _ = grift.Namespace("db", func() {
 
 		treasuresToBury := []models.Treasure{}
 
-		err := models.DB.RawQuery("SELECT * from treasures").All(&treasuresToBury)
+		err := models.DB.RawQuery("SELECT * from treasures ORDER BY created_at asc").All(&treasuresToBury)
```

Comment:
nit: ASC

Reply:
Okay.  I'll go ahead and do all of them.